File: app/src/main/java/com/wolcano/musicplayer/music/widgets/ScaledImageView.java
package com.wolcano.musicplayer.music.widgets;

import android.content.Context;
import android.graphics.Bitmap;
import android.graphics.Canvas;
import android.view.View;

public class ScaledImageView extends View {
    public Bitmap imageBitmap;
    public ScaledImageView(Context context) {
        super(context);
    }
    @Override
    protected void onDraw(Canvas canvas) {
        super.onDraw(canvas);

        if (imageBitmap != null)
            canvas.drawBitmap(imageBitmap, 0, 0, null);
    }
}

File: app/src/main/java/com/wolcano/musicplayer/music/widgets/ScaledImageView.kt
package com.wolcano.musicplayer.music.widgets

import android.content.Context
import android.graphics.Bitmap
import android.graphics.Canvas
import android.view.View

class ScaledImageView(context: Context?) : View(context) {

    var imageBitmap: Bitmap? = null

    override fun onDraw(canvas: Canvas) {
        super.onDraw(canvas)

        if (imageBitmap != null) {
            canvas.drawBitmap(imageBitmap!!, 0f, 0f, null)
        }
    }

}